controllers/twophase/types.go

```diff
@@ -61,206 +59,64 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-			}
-		}
-
-		chaos.SetNextRecover(time.Time{})
-
-		status.Experiment.EndTime = &metav1.Time{
-			Time: time.Now(),
-		}
-		status.Experiment.Phase = v1alpha1.ExperimentPhaseWaiting
-		status.FailedMessage = emptyString
-	} else if (status.Experiment.Phase == v1alpha1.ExperimentPhaseFailed ||
-		status.Experiment.Phase == v1alpha1.ExperimentPhasePaused) &&
-		!chaos.GetNextRecover().IsZero() && chaos.GetNextRecover().After(now) {
-		// Only resume/retry chaos in the case when current round is not finished,
-		// which means the current time is before recover time. Otherwise we
-		// don't resume the chaos and just wait for the start of next round.
-
-		r.Log.Info("Resuming/Retrying")
-
-		dur := chaos.GetNextRecover().Sub(now)
-		if err = applyAction(ctx, r, req, dur, chaos); err != nil {
-			updateFailedMessage(ctx, r, chaos, err.Error())
-			return ctrl.Result{Requeue: true}, err
-		}
-
-		status.FailedMessage = emptyString
-	} else if chaos.GetNextStart().Before(now) {
-		r.Log.Info("Starting")
-
-		tempStart, err := utils.NextTime(*chaos.GetScheduler(), now)
-		if err != nil {
-			r.Log.Error(err, "failed to calculate the start time")
-			updateFailedMessage(ctx, r, chaos, err.Error())
-			return ctrl.Result{}, err
-		}
-
-		tempRecover := now.Add(*duration)
-		if tempStart.Before(tempRecover) {
-			err := fmt.Errorf("nextRecover shouldn't be later than nextStart")
-			r.Log.Error(err, "Then recover can never be reached.", "scheduler", *chaos.GetScheduler(), "duration", *duration)
-			updateFailedMessage(ctx, r, chaos, err.Error())
-			return ctrl.Result{}, err
-		}
-
-		if err = applyAction(ctx, r, req, *duration, chaos); err != nil {
-			updateFailedMessage(ctx, r, chaos, err.Error())
-			return ctrl.Result{Requeue: true}, err
-		}
-
-		nextStart, err := utils.NextTime(*chaos.GetScheduler(), status.Experiment.StartTime.Time)
-		if err != nil {
-			r.Log.Error(err, "failed to get the next start time")
-			return ctrl.Result{}, err
-		}
-		nextRecover := status.Experiment.StartTime.Time.Add(*duration)
-
-		chaos.SetNextStart(*nextStart)
-		chaos.SetNextRecover(nextRecover)
-		status.FailedMessage = emptyString
-	} else {
-		r.Log.Info("Waiting")
-
-		nextStart, err := utils.NextTime(*chaos.GetScheduler(), status.Experiment.StartTime.Time)
-		if err != nil {
-			r.Log.Error(err, "failed to get next start time")
-			return ctrl.Result{}, err
-		}
-		nextTime := chaos.GetNextStart()
-
-		// if nextStart is not equal to nextTime, the scheduler may have been modified.
-		// So set nextStart to time.Now.
-		if nextStart.Equal(nextTime) {
-			if !chaos.GetNextRecover().IsZero() && chaos.GetNextRecover().Before(nextTime) {
-				nextTime = chaos.GetNextRecover()
-			}
-			duration := nextTime.Sub(now)
-			r.Log.Info("Requeue request", "after", duration)
+		targetPhase = v1alpha1.ExperimentPhaseFinished
+	}
 
-			return ctrl.Result{RequeueAfter: duration}, nil
-		}
+	r.Log.Info("decide target phase", "target phase", targetPhase)
```

Comment:
how about adding the chaos's name in all the log of this function `Reconcile `?

Reply:
added in the name